doc/source/parameters/cfd/fem.rst

```diff
@@ -0,0 +1,41 @@
+Finite element interpolation (FEM)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This subsection specifies the characteristics of the finite element method used in the simulations. The interpolation orders of velocity, pressure and other physics are specified. An additional option is also present to enable to use of high-order mapping throughout the entire mesh. At the moments, all variables are interpolated using Lagrange elements. On quad/hex meshes, `FE_Q <https://www.dealii.org/current/doxygen/deal.II/classFE__Q.html>`_ elements are used whereas on simplex meshes (triangles, tetrahedron), `FE_P <https://www.dealii.org/current/doxygen/deal.II/classFE__SimplexP.html>`_ elements are used.
+
+
+.. code-block:: text
+
+   subsection FEM
+     # interpolation order pressure
+     set pressure order    = 1
+  
+     # interpolation order velocity
+     set velocity order    = 1
+ 
+     # interpolation order temperature
+     set temperature order    = 1
+ 
+     # interpolation order tracer
+     set tracer order    = 1
+   
+     # interpolation order vof
+     set VOF order  =1
+ 
+     # Apply high order mapping everywhere
+     set qmapping all      = false
+   end
+
+* ``velocity order`` specifies the interpolation order for velocity.
```

Comment:
This is a suggestion but it's very unnecessary, but maybe we could link this to a fem explanation with the lagragian polynomial?

Reply:
We already link to the documentation for the classes of polynomial, we could definitely add a page that gives some basic information about the finite element method and interpolation!